recipes/certify/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20201114":
```

Comment:
There's a more recent version 20211012, is there a reason to only this one? Or maybe future PR to add something more recent